docs/operate/get-started/setup-micro/_index.md

```diff
@@ -0,0 +1,404 @@
+---
+linkTitle: "Set up an ESP32"
+title: "Set up an ESP32"
+weight: 25
+layout: "docs"
+type: "docs"
+images: ["/installation/thumbnails/install.png"]
+imageAlt: "Install Viam"
+no_list: false
+description: "Install the lightweight version of the software that drives hardware and connects your device to the cloud."
+prev: "/operate/get-started/basics/"
+next: "/operate/get-started/other-hardware/micro-module/"
+aliases:
+  - /installation/microcontrollers/
+  - /installation/prepare/microcontrollers/
+  - /installation/prepare/microcontrollers/
+  - /build/micro-rdk/
+  - /get-started/installation/microcontrollers/
+  - /installation/viam-micro-server-setup/
+  - /operate/reference/viam-micro-server/viam-micro-server-troubleshooting/
+---
+
+Viam maintains a [lightweight version of `viam-server`](/operate/reference/viam-micro-server/) for microcontrollers.
+
+## Supported microcontrollers
+
+ESP32 microcontrollers must have at least 2 cores, 384kB SRAM, 2MB PSRAM and 8MB flash to work with Viam.
```

Comment:
```suggestion
To work with Viam's micro-RDK, ESP32 microcontrollers must have at least:

- 2 cores
- 384kB SRAM
- 2MB PSRAM
- 8MB flash
```

[s] I think there are enough items here to warrant an unordered list to make things more scannable. Additionally, an ordered list removes potential reader confusion around the scoping of the 'at least' modifier.

[opt] I think this sentence buries the 'to work with Viam' lede a bit too deep. Otherwise, it's hard to know the point of this sentence when you first start reading it!

Reply:
Fair! We originally smushed the list into a single line to keep the quickstart further above the fold, but I think it's a tossup which is more important here